src/cockpit/389-console/src/lib/monitor/replMonConflict.jsx

```diff
@@ -430,7 +430,7 @@ export class ReplMonConflict extends React.Component {
                             <Tooltip
                                 content={
                                     <div>
-                                        {_("When a <b>Delete</b> operation is replicated and the consumer server finds that the entry to be deleted has child entries, the conflict resolution procedure creates a \"<i>glue entry</i>\" to avoid having orphaned entries in the database.  In the same way, when an <b>Add</b> operation is replicated and the consumer server cannot find the parent entry, the conflict resolution procedure creates a \"<i>glue entry</i>\", representing the \"parent entry\", so that the new entry is not an orphaned entry.  You can choose to convert the glue entry, or remove the glue entry and all its child entries.")}
+                                        {_("When a Delete operation is replicated and the consumer server finds that the entry to be deleted has child entries, the conflict resolution procedure creates a \"glue entry\" to avoid having orphaned entries in the database.  In the same way, when an <b>Add</b> operation is replicated and the consumer server cannot find the parent entry, the conflict resolution procedure creates a \"glue entry\", representing the \"parent entry\", so that the new entry is not an orphaned entry.  You can choose to convert the glue entry, or remove the glue entry and all its child entries.")}
```

Comment:
```suggestion
                                        {_("When a Delete operation is replicated and the consumer server finds that the entry to be deleted has child entries, the conflict resolution procedure creates a \"glue entry\" to avoid having orphaned entries in the database.  In the same way, when an Add operation is replicated and the consumer server cannot find the parent entry, the conflict resolution procedure creates a \"glue entry\", representing the \"parent entry\", so that the new entry is not an orphaned entry.  You can choose to convert the glue entry, or remove the glue entry and all its child entries.")}
```